torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1200,6 +1200,39 @@ def sample_inputs_svd(op_info, device, dtype, requires_grad=False):
 def sample_inputs_linalg_svd(op_info, device, dtype, requires_grad=False):
     return _sample_inputs_svd(op_info, device, dtype, requires_grad, is_linalg_svd=True)
 
+def sample_inputs_eig(op_info, device, dtype, requires_grad=False):
+    eigvecs = make_tensor((S, S), device=device, dtype=dtype,
+                          low=None, high=None)
+    eigvals = make_tensor((S,), device=device, dtype=dtype,
+                          low=None, high=None)
+    input = (eigvecs * eigvals.unsqueeze(-2)) @ eigvecs.inverse()
+    input.requires_grad_(requires_grad)
+
+    def process_input(eigpair):
```

Comment:
But this functions processes the output of `torch.eig`

Reply:
Yes, the OpInfo test is for `torch.eig`.

Reply:
The name of the function is `process_input`, but it should be `process_output`.